client/jetpack-connect/test/plans.js

```diff
@@ -14,10 +14,10 @@ import React from 'react';
 import PlansWrapper, { getSitePlans, SELECTED_SITE, SITE_PLAN_PRO } from './lib/plans';
 import { PLAN_JETPACK_BUSINESS } from 'lib/plans/constants';
 
-jest.mock( 'components/data/query-plans', () => require( 'components/empty-component' ) );
-jest.mock( 'components/data/query-site-plans', () => require( 'components/empty-component' ) );
-jest.mock( 'jetpack-connect/happychat-button', () => require( 'components/empty-component' ) );
-jest.mock( 'my-sites/plan-features', () => require( 'components/empty-component' ) );
+jest.mock( 'components/data/query-plans', () => 'components--data--query-plans' );
```

Comment:
Yep, seems a good convention for the tag name.

Other possibilities that come to mind:
`components_data_query-plans`
`components__data__query-plans`

Reply:
[`-` _must_ be included](https://w3c.github.io/webcomponents/spec/custom/#valid-custom-element-name), so `--` seemed to be a good path separator replacement.